src/react/providers/chat-client-provider.tsx

```diff
@@ -37,6 +37,12 @@ export const ChatClientProvider = ({ children, client }: ChatClientProviderProps
   const value: ChatClientContextValue = React.useMemo(() => {
     // Set the internal useReact option to true to enable React-specific agent.
     (client as unknown as { addReactAgent(): void }).addReactAgent();
+
+    const uiKitVersion = (globalThis as Record<string, unknown>).__ABLY_CHAT_REACT_UI_COMPONENTS_VERSION__;
```

Comment:
slightly cleaner to make a global.d.ts file with:
```typescript
declare global {
  interface Window {
    __ABLY_CHAT_REACT_UI_COMPONENTS_VERSION__?: string;
  }
}
```

Reply:
you can use the same trick to declare the `addAgentWithVersion` method internally too :)